components/usergroup/organizer.tsx

```diff
@@ -28,13 +28,15 @@ export const Organizer = ({
           className="size-17 overflow-hidden rounded-full"
           data-tina-field={tinaField(data, "name")}
         >
-          <Image
-            alt={`Picture of ${data?.name ?? "Organizer"}`}
-            src={data?.profileImg ?? ""}
-            height={68}
-            width={68}
-            className="w-17"
-          />
+          {data.profileImg && (
+            <Image
+              alt={`Picture of ${data?.name ?? "Organizer"}`}
+              src={data?.profileImg ?? ""}
+              height={68}
+              width={68}
```

Comment:
can we add a fallback for image if it returns 404? i.e onError prop for Image component

Reply:
Done - I added a fallback so that if the image breaks it will be replaced with https://www.ssw.com.au/images/thumbs/avatar-thumbnail.png